htdocs/web_portal/views/downtime/add_downtime.php

```diff
@@ -497,7 +498,19 @@ function getSitesServices(){
         var siteId=$('#Select_Sites').val();
         if(siteId != null){ //If the user clicks on the box but not a specific row there will be no input, so catch that here
             $('#chooseEndpoints').empty(); //Remove any previous content from the endpoints select list
-            $('#chooseServices').load('index.php?Page_Type=Downtime_view_endpoint_tree&site_id='+siteId,function( response, status, xhr ) {
+            
+            let DVET_Path;
+            const getServiceIDFromQueryParams = new URL(this.location.href).searchParams.get('se');
+
+            if (getServiceIDFromQueryParams) {
+                // NOTE: If you use template literal for string interpolation, i.e., `${}`. It has to be in one line.
+                DVET_Path = 'index.php?Page_Type=Downtime_view_endpoint_tree' +
```

Comment:
Hi @gregcorbett , Regarding the issue #https://github.com/GOCDB/gocdb/pull/439#issuecomment-1534999143.

It seems the jQuery Engine is having an issue with the template literal. It is wrongly interpreting the string interpolation if it is multi-line i.e., `${}`. I have changed the code to multi-line because code climate identified a problem with the space limit for a statement. It was working before when it was a single line statement for template literal. I have modified the code to support multi-line. Thus, resolving the space issue. I spend some time doing testing and figured. It should be working fine now.